scss/detail.scss

```diff
@@ -0,0 +1,33 @@
+.book {
+	color: $black;
+
+	&__title {
+		margin: 0 0 52px 0;
+		h1 {
+			font-size: 39px;
+			font-weight: 700;
+		}
+
+		span {
+			font-size: 30px;
+			font-weight: bold;
+		}
+	}
+
+	&__detail ul li{
+		margin: 0 0 36px 0;
+		h2 {
+			font-size: 22px;
```

Comment:
If your `h2` and `span` tags have the same `font-size`, try set in same selector block.